docs/variants/qemu_q35/releases.md

```diff
@@ -3,15 +3,39 @@
 Following Release Notes describe status of Open Source Firmware development for
 QEMU Q35 (Emulator).
 
-## v0.1.0 - 2023-01-20
+## v0.1.0 - 2023-10-13
 
 ### Added
 
-- Enabled TPM Support.
-- Secure Boot Support for Q35 machine.
-- Option to reset Secure Boot keys to defaults.
-- Dasharo System Features.
-- Dasharo setup password.
-- OPAL disk password support.
-- SATA disk password support.
-- SMM Feature enabled.
+- Configurable boot order
```

Comment:
I have gathered the features we had + enabled the ones not enabled in: https://github.com/Dasharo/edk2/pull/92

The thing is, many of them are just menus with no proper backed, as far as I understand.

Moreover, we will not test them properly right now (even if they would work), as in this v0.1.0 the main goal would be to have just menus and being able to test the menus (for correct strings, default values, moving around in the menus).

What can be "properly" tested right now is:
- custom boot menu keys
- UEFI shell

At least I have confirmed those 2 tests from our suite can pass.

Reply:
@macpijan that is fine if we clearly say that in the release notes. I guess one of the big things should be UEFI Secure Boot and DTS validation. For the further one, I would expect to create the correct backend on the QEMU side to handle various cases correctly.

Creating the correct backend for other features does not make sense for us now, but there is huge potential in confidential computing, which could fuel efforts to improve what we do for QEMU Q35, it probably would mean a slightly different goal and maybe QEMU target but fundamental operating principles would be the same.

Reply:
@pietrushnic see my proposal of dividing these in: https://github.com/Dasharo/docs/pull/678/commits/45e294d357c78168eac9c27ddd09818f09dc4794

Reply:
@macpijan I love it. Your proposal is practical, and we clearly see where we are. What is more, we can build further roadmap on this information. IMHO it it good enough to close this discussion.